Server/VikingVault.API/Controllers/CardController.cs

```diff
@@ -13,19 +15,18 @@
 
 namespace VikingVault.API.Controllers
 {
-    [Authorize]
+    [Route("[controller]")]
     [ApiController]
     public class CardController : ControllerBase
     {
-        private IAttachCardService _attachCardService;
+        private ICardService _attachCardService;
```

Comment:
Why do you name the variable _attachCardService when the type is actually just cardService?